crates/sui-core/src/authority.rs

```diff
@@ -122,6 +122,7 @@ pub struct AuthorityMetrics {
     pub follower_connections: IntCounter,
     pub follower_connections_concurrent: IntGauge,
 
+    // TODO: consolidate these into GossipMetrics
```

Comment:
Lets make an issue, tag it as "good first issue", for people joining the team and looking for ways to get into the codebase.

Reply:
https://github.com/MystenLabs/sui/issues/3926